bikespace_frontend/src/components/submission/submission-form/SubmissionForm.tsx

```diff
@@ -65,21 +64,6 @@ export function SubmissionForm() {
     comments: comments,
   };
 
-  useEffect(() => {
-    navigator.geolocation.getCurrentPosition(
-      position => {
-        setLocation({
-          latitude: position.coords.latitude,
-          longitude: position.coords.longitude,
-        });
-        setLocationLoaded(true);
-      },
-      () => {
-        setLocationLoaded(true);
-      }
-    );
-  }, []);
-
```

Comment:
I think we should actually keep this useEffect to geolocate the user (without the locationLoaded updates). If we rely on the map geolocate control, it always starts from city hall and then animates moving to the user's location. Whereas this method loads the map where the user is right away. The only downside is that it's a bit confusing to ask for location permissions on page load instead of when this question loads, but maybe there's a way to fix that later.